Collaboration2/Collaboration2/Scenes/Weather Page/WeatherPageViewModel.swift

```diff
@@ -36,4 +40,35 @@ class WeatherPageViewModel: ObservableObject {
     func imageURL(url: String) -> URL? {
         URL(string: url)
     }
+    
+    func getHumidity() -> Int {
```

Comment:
I'm not sure abut this but maybe it better to rewrite this functions as lazy computed properties